lib/core/src/Cardano/Wallet/Primitive/AddressDerivation.hs

```diff
@@ -704,6 +704,11 @@ class WalletKey (key :: Depth -> * -> *) where
         :: key depth raw
         -> raw
 
+    -- | Lift 'XPrv' or 'XPub' to 'WalletKey'.
+    liftRawKey
+        :: raw
+        -> key depth raw
```

Comment:
Is this used anywhere? This is arguably an unsafe backdoor which I'd rather not have in the module interface :thinking:

Reply:
ah, yes. introduced to support version in which scriptHash is associated with our keys.Right now we store indices and this is not needed anymore.will remove it

Reply:
removed in https://github.com/input-output-hk/cardano-wallet/pull/2308/commits/2ffef82dd2538db1123289ca95079056fca48894

Reply:
hmm.in the end need this method,so reverted -> https://github.com/input-output-hk/cardano-wallet/pull/2308/commits/bbdfbc8e50f6bc91fbb0be153a4a9467ea308ab5

Reply:
here https://github.com/input-output-hk/cardano-wallet/pull/2308/files#diff-843eab8af2dd1a2491a9fbff3af14a2dbafac7ea475624c0e4840af393370fb7R503
for
```
deriveVerificationKey
    :: (SoftDerivation k, WalletKey k)
    => k 'AccountK XPub
    -> Index 'Soft 'ScriptK
    -> k 'ScriptK XPub
deriveVerificationKey accXPub =
    liftRawKey . getRawKey . deriveAddressPublicKey accXPub MultisigScript . coerce
```